LEFT_HAND:
- ACTION_NAME: 张开
  ACTION_POS:
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 10
  - 100
  - 180
  - 240
  - 245
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
  - 255
RIGHT_HAND:
- ACTION_NAME: 张开
  ACTION_POS:
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 10.0
  - 100.0
  - 180.0
  - 240.0
  - 245.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
- ACTION_NAME: 握拳
  ACTION_POS:
  - 69.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
  - 151.0
  - 10.0
  - 100.0
  - 180.0
  - 240.0
  - 14.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 109.0
  - 0.0
  - 0.0
  - 0.0
  - 0.0
- ACTION_NAME: '666'
  ACTION_POS:
  - 255.0
  - 255.0
  - 33.0
  - 255.0
  - 255.0
  - 255.0
  - 10.0
  - 100.0
  - 180.0
  - 240.0
  - 245.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
  - 255.0
